client/src/screens/home.screen.js

```diff
@@ -1,108 +1,160 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import {
-  FlatList,
-  ActivityIndicator,
-  Button,
-  Image,
-  StyleSheet,
-  Text,
-  TouchableHighlight,
-  View,
-} from 'react-native';
-import { graphql, compose } from 'react-apollo';
-import moment from 'moment';
+import { FlatList, StyleSheet, Text, View } from 'react-native';
 import Icon from 'react-native-vector-icons/FontAwesome';
 import { connect } from 'react-redux';
 
-import CURRENT_USER_QUERY from '../graphql/current-user.query';
+import { extendAppStyleSheet } from './style-sheet';
 
-const styles = StyleSheet.create({
-  container: {
-    backgroundColor: 'white',
-    flex: 1,
-  },
-  loading: {
-    justifyContent: 'center',
-    flex: 1,
-  },
-  groupContainer: {
-    flex: 1,
+const styles = extendAppStyleSheet({
+  sectionHeading: {
+    alignItems: 'center',
+    backgroundColor: '#EEE',
+    borderBottomWidth: 1,
+    borderColor: '#CCC',
     flexDirection: 'row',
+    justifyContent: 'space-between',
+    padding: 10
+  },
+  availabilityRow: {
     alignItems: 'center',
-    backgroundColor: 'white',
-    borderBottomColor: '#eee',
     borderBottomWidth: 1,
-    paddingHorizontal: 12,
-    paddingVertical: 8,
+    borderColor: '#CCC',
+    flexDirection: 'row',
+    padding: 6
   },
-  groupName: {
-    fontWeight: 'bold',
-    flex: 0.7,
+  indicator: {
+    backgroundColor: 'yellow',
+    borderColor: 'black',
+    borderRadius: 10,
+    borderWidth: 1,
+    height: 20,
+    marginLeft: 6,
+    marginRight: 12,
+    width: 20
   },
-  groupTextContainer: {
-    flex: 1,
-    flexDirection: 'column',
-    paddingLeft: 6,
+  available: {
+    backgroundColor: '#77D353',
+    borderColor: '#44A020'
   },
-  groupText: {
-    color: '#8c8c8c',
+  unavailable: {
+    backgroundColor: '#F95F62',
+    borderColor: '#930000'
   },
-  groupImage: {
-    width: 54,
-    height: 54,
-    borderRadius: 27,
+  eventCard: {
+    backgroundColor: '#EEE',
+    borderColor: '#CCC',
+    borderWidth: 1,
+    margin: 10,
+    marginBottom: 0,
+    padding: 10
   },
-  groupTitleContainer: {
+  eventHeading: {
     flexDirection: 'row',
+    justifyContent: 'space-between'
   },
-  groupLastUpdated: {
-    flex: 0.3,
-    color: '#8c8c8c',
-    fontSize: 11,
-    textAlign: 'right',
-  },
-  groupUsername: {
-    paddingVertical: 4,
-  },
-  header: {
-    alignItems: 'flex-start',
-    padding: 6,
-    borderColor: '#eee',
-    borderBottomWidth: 1,
-  },
-  warning: {
-    textAlign: 'center',
-    padding: 12,
+  eventAgo: {
+    color: '#333',
+    fontSize: 10
   },
+  eventUrgent: {
+    borderColor: 'red'
+  }
 });
 
 class Home extends Component {
   static navigationOptions = {
     title: 'Home',
-    tabBarIcon: ({ tintColor}) => <Icon size={30} name={'home'} color={tintColor} />
+    tabBarIcon: ({ tint }) => <Icon size={48} name='home' color={tint} />
```

Comment:
reverting this size increase cause it looks dumb on iOS. Clearly we just need to handle the styles differently on iOS vs android but I have more important things to work on right now. Feel free to take it on if you'd like.

Reply:
Good spot, the size change wasn't deliberate - I was probably trying to fiddle with the power icon size.

Reply:
The variable name change here actually broke the tint colour being applied as well. This is a JS unpacking construct so the name needs to match what is in the argument passed to this func. There's ways to get around this but you can google.